test/wrapper_test.dart

```diff
@@ -87,6 +87,9 @@ const TO_STRING_INVOCATION = const SyntheticInvocation(
 class IterableNSM extends NSM implements Iterable, Set, List, Queue {
   IterableNSM(action(Invocation i)) : super(action);
   toString() => super.noSuchMethod(TO_STRING_INVOCATION) as String;
+
+  Null cast<T>() => null;
+  Null retype<T>() => null;
```

Comment:
Make these methods abstract. The class uses `noSuchMethod` to handle (and record) everything in the `Iterable` interface, so it shouldn't need implementations. I can see that the return type needs fixing here since it must be both `List` and `Set` (and I should proabably rewrite the test to not use the same class for everything :)

Reply:
Done.